run_test.sh

```diff
@@ -148,3 +148,40 @@ set -e
 npm i
 npm t 
 "
+
+rm -rf /tmp/holo-flake
+cd /tmp
+
+hc-scaffold web-app holo-flake --template vue
+cd holo-flake
+
+nix develop --command bash -c "
+set -e
+# Check if holo-dev-server is in the path
+if command -v holo-dev-server >/dev/null 2>&1; then
+  echo 'holo-dev-server is available in the PATH while it shouldn\'t'
```

Comment:
The apostrophe doesn't appear to be correctly escaped, might be what's causing CI to fail

```suggestion
  echo 'holo-dev-server is available in the PATH while it shouldn'\''t'
```